platform-tooling-support-tests/src/test/java/platform/tooling/support/tests/StandaloneTests.java

```diff
@@ -83,6 +86,57 @@ void listAllObservableEngines() {
 			result.getOutput("out").lines());
 	}
 
+	@Test
+	void printVersionViaJar() {
+		var result = Request.builder() //
+				.setTool(new Java()) //
+				.setProject("standalone") //
+				.addArguments("-jar", MavenRepo.jar("junit-platform-console-standalone")) //
+				.addArguments("--version") //
+				.build() //
+				.run();
+
+		assertEquals(0, result.getExitCode(), () -> getExitCodeMessage(result));
+
+		var version = Helper.version("junit-platform-console");
+		assertLinesMatch("""
+				JUnit Platform Console Launcher %s
+				JVM: .*
+				OS: .*
+				""".formatted(version).lines(), //
+			result.getOutput("out").lines());
+	}
+
+	@Test
+	void printVersionViaModule() {
+		var junitJars = Stream.of("junit-platform-console", "junit-platform-reporting", "junit-platform-engine",
+			"junit-platform-launcher", "junit-platform-commons") //
+				.map(MavenRepo::jar);
+		var thirdPartyJars = Stream.of(ThirdPartyJars.find("org.opentest4j", "opentest4j"));
+		var modulePath = Stream.concat(junitJars, thirdPartyJars) //
+				.map(String::valueOf) //
+				.collect(joining(File.pathSeparator));
+		var result = Request.builder() //
+				.setTool(new Java()) //
+				.setProject("standalone") //
+				.addArguments("--module-path", modulePath) //
+				.addArguments("--add-modules", "org.junit.platform.console") //
```

Comment:
```suggestion
```

Line can removed. The launched module is always added to the module graph (as a root module).